be/src/pipeline/exec/group_commit_block_sink_operator.h

```diff
@@ -30,7 +31,7 @@ class GroupCommitBlockSinkOperatorBuilder final
     GroupCommitBlockSinkOperatorBuilder(int32_t id, DataSink* sink)
             : DataSinkOperatorBuilder(id, "GroupCommitBlockSinkOperator", sink) {}
 
-    OperatorPtr build_operator() override;
+    inline OperatorPtr build_operator() override;
```

Comment:
while add inline？ seems unless

Reply:
build_operator() implemented in group_commit_block_sink_operator.h. Without inline, there will be compile error "multiple definitions".